packages/salesforcedx-vscode-core/src/commands/functions/functionService.ts

```diff
@@ -110,6 +110,19 @@ export class FunctionService {
     return this.startedExecutions.size > 0;
   }
 
+  /**
+   * Returns the debugType of the first of the startedExecutions as a way to determine the language
```

Comment:
Nit pick - Currently we run all the commands with the assumption that the user has only running function.

Reply:
Bring on the nit picks - I'll learn more that way! 👍 I'm a little unclear on the recommended change, can you clarify? Is it just changing the comment, or something in the method?

Reply:
Just changing the comment.